src/book/02-projects/03-hardware/01-overview.mdx

```diff
@@ -6,9 +6,33 @@ description: Overview and specification of the hardware used by NUbots.
 slug: /projects/hardware/overview
 ---
 
-The platform used is the NUgus, based off the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling. Because of this, the components of the robot were split to allow for the size of the printer bed and the printer volume. The torso was also modified to have open shoulder covers for shoulder servo cable access.
+NUbots uses the NUgus hardware platform, a modification of the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling.
 
-![Coronal image of the NUgus robot](../../../images/front-robot.png)
-![Sagittal image of the NUgus robot](../../../images/side-robot.png)
+![Coronal and sagittal image of the NUgus robot](../../../images/nugus.png 'Coronal and sagittal image of the NUgus robot')
```

Comment:
@tayloryoung6396 Any chance we could get a higher resolution transparent version of these images?

Reply:
Probably